package.json

```diff
@@ -4,17 +4,20 @@
   "private": true,
   "homepage": ".",
   "scripts": {
-    "build": "node dev-scripts/copy-localhost-data.js && react-scripts build",
+    "build": "react-scripts build",
+    "ci:cypress": "yarn start-test \"yarn concurrently 'yarn eth:node' 'yarn ci:start'\" '8545|3000' 'yarn eth:deploy-dao-contracts-on-hardhat && yarn cypress:run'",
+    "ci:start": "FORCE_COLOR=true yarn start | cat",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "eject": "react-scripts eject",
-    "eth:compile-contracts": "cd hardhat && npx hardhat compile",
+    "eth:deploy-dao-contracts-on-hardhat": "node dev-scripts/deploy-dao-contracts-on-hardhat.js",
     "eth:node": "cd hardhat && npx hardhat node --network hardhat",
+    "eth:prepare-dao-contracts-for-hardhat": "node dev-scripts/prepare-dao-contracts-for-hardhat.js",
```

Comment:
I would drop the "on-hardhat" and "for-hardhat" parts of the script names but don't feel strongly here

Reply:
I'd keep it this way, because there is another command which "fixes" the DAO contracts just for typechain.